pkg/jobqueue/dbjobqueue/dbjobqueue.go

```diff
@@ -94,6 +94,12 @@ const (
 		WHERE id = $1 AND finished_at IS NULL
 		RETURNING type, started_at`
 
+	sqlFailJob = `
```

Comment:
Actually this is fine to have a separate query.

Reply:
From how I understood the behavior of the DB, the token, started_at, finished_at have to be set in order to get to a "failed" state where no jobs have been started (our case here)
otherwise it might appear in "requeue" and other queries

Reply:
Hopefully I understood this correctly